packages/core/src/Synchronizer.ts

```diff
@@ -543,7 +587,7 @@ export class Synchronizer extends EventEmitter {
         db.create('Attestation', {
             epoch: _epoch,
             epochKey: _epochKey.toString(16),
-            // index: attestIndex,
+            index: +`${event.blockNumber}${event.transactionIndex}${event.logIndex}`,
```

Comment:
I think this actually breaks if `blockNumber` or tx index is a variable length (e.g. block number `400`  and tx index `100` would be more than block number `1000` and tx index `99`)